configs/azure_iot_sdksConfig.cmake

```diff
@@ -1,6 +1,19 @@
 #Copyright (c) Microsoft. All rights reserved.
 #Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+message("azure_iot_sdksConfig.cmake: Line 4: ${CMAKE_CURRENT_LIST_DIR}")
+
+include(CMakeFindDependencyMacro)
+find_dependency(unofficial-parson)
```

Comment:
This will add a hard dependency on vcpkg, which is the only provider of the "unofficial-parson" package.